test/kb_SPAdes_server_test.py

```diff
@@ -700,7 +700,7 @@ def test_metagenome(self):
                }],
              'md5': '08d0b92ce7c0a5e346b3077436edaa42',
              'fasta_md5': 'ca42754da16f76159db91ef986f4d276'
-             }, dna_source='metagenomic')
+             }, min_contig_len=500, dna_source='metagenomic')
```

Comment:
Yes, this is a mystery to me as well. Most of the tests are assertions on output meta-data like number of output objects created, number of provenance entries etc. But there is one assertion on the md5 hash of the assembly in the report. I would have expected this value to change with varying min_contig_len but it does not. I will need to spend some time understanding the application in order to debug this.